client/my-sites/patterns/components/pattern-library/index.tsx

```diff
@@ -164,6 +174,12 @@ export const PatternLibrary = ( {
 		};
 	}, [] );
 
+	useEffect( () => {
+		if ( navigationElementRef.current && section && ! searchTerm ) {
+			scrollToSection( navigationElementRef.current, 'instant' );
+		}
```

Comment:
We should check that the hash actually matches `#readymade-templates-section` before scrolling to the the RMT section.